src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -47,8 +48,13 @@ public class IntakeSubsystem extends SubsystemBase {
 	private final DigitalInput indexSensor;
 	private final DigitalInput feederSensor;
 
+	private final SparkLimitSwitch intakeFrontSensor;
+	private final SparkLimitSwitch intakeBackSensor;
+	private final SparkLimitSwitch intakeLeftSensor;
+	private final SparkLimitSwitch intakeRightSensor;
```

Comment:
We should probably log these to help debug any issues in the future.

Reply:
added them to shuffleboard